### PR TITLE
doc: add more companies to power-by page 

### PR DESCRIPTION
**Pull request by Yiyiyimu**
### What this PR does / why we need it:
Update since #1317. Need to update image @moonming. 
Also, do we need to sync the image here and [README](https://github.com/apache/apisix#who-uses-apisix)?

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: doc/powered-by.md [hunk 1]
```diff
@@ -67,4 +74,4 @@ Using APISIX as a gateway, it uses the functions of current limiting, speed limi
 Using APISIX as a Web ACL gateway to deal with backend OA systems traffic.
 
 ## HelloTalk, Inc.
-Using Apache APISIX as an API gateway to manage all API and SSL certificates in test\dev\CMS environment.
```
Comment (nic-chen) [inline doc/powered-by.md:70]: do not remove `Apache`

Comment (Yiyiyimu) [reply doc/powered-by.md:70]: The name format of these user cases are not consistent. Or I could add Apache to all lines.

**Comment by moonming** [comment]
+1 to add `Apache` to all lines.

Thanks,
Ming Wen
Twitter: _WenMing


Shuyang Wu <notifications@github.com> 于2020年8月25日周二 下午4:51写道：

**Comment by Yiyiyimu** [comment]
> +1 to add `Apache` to all lines. 

Updated

**Review by membphis — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
I think we need to update the company list image sync the image here and in README before merge this PR

**Comment by moonming** [comment]
I will update the user wall later.

Thanks,
Ming Wen
Twitter: _WenMing


Shuyang Wu <notifications@github.com> 于2020年8月26日周三 下午1:55写道：

**Pull request merged**